pkg/eksd/installer.go

```diff
@@ -24,24 +26,26 @@ type EksdInstallerClient interface {
 type Installer struct {
 	client  EksdInstallerClient
 	retrier *retrier.Retrier
+	reader  *filereader.Reader
 }
 
 func NewEksdInstaller(client EksdInstallerClient) *Installer {
 	return &Installer{
 		client:  client,
 		retrier: retrier.NewWithMaxRetries(maxRetries, backOffPeriod),
+		reader:  filereader.NewReader(),
```

Comment:
Can we inject this as a dependency?
If not, the `clusterSpec` has a method to read manifests